cob_hardware_config/xacrodisplay.launch

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<launch>
+  <arg name="robot" default="$(optenv ROBOT !!NO_ROBOT_SET!!)"/>
+  <arg name="gui" default="false" />
+  <arg name="rvizconfig" default="$(find urdf_tutorial)/urdf.rviz" />
```

Comment:
I vote for using the existing rviz configs. it does not hurt if there are laser scanners and cameras defined in the config. if we introduce a new rviz config thats one more file we have to maintain without gaining mayor benefits.